nautobot/dcim/signals.py

```diff
@@ -96,7 +96,8 @@ def assign_virtualchassis_master(instance, created, **kwargs):
     if created and instance.master:
         master = Device.objects.get(pk=instance.master.pk)
         master.virtual_chassis = instance
-        master.vc_position = 1
+        if not instance.master.vc_position:
```

Comment:
I think this should actually be `if instance.master.vc_position is None` as `vc_position` can be legitimately set to `0` and we wouldn't want to change that if so.

(Consider either changing `test_master_device_vc_assignment` to use `0` instead of `3`, or adding another test case specifically for that possibility).